modules/ROOT/pages/index.adoc

```diff
@@ -185,6 +185,32 @@ Anypoint Monitoring is the standard method of monitoring Mule application and AP
 
 |===
 
+== Use Cases
+
+The following use cases provide examples of how you can use Anypoint Monitoring.
+
+* Understand performance-related KPIs+
```

Comment:
KPIs+ > **KPIs +** (there's a missing space that prevents the **+** from creating a new line)

Reply:
```suggestion
* Understand performance-related KPIs +
```